runtime/pdo/pdo.cpp

```diff
@@ -8,7 +8,9 @@
 #include "runtime/array_functions.h"
 #include "runtime/pdo/pdo.h"
 #include "runtime/pdo/pdo_statement.h"
+#ifdef PDO_MYSQL_SUPPORT
 #include "runtime/pdo/mysql/mysql_pdo_driver.h"
+#endif
```

Comment:
Why is it needed?
This header contains only declarations

Reply:
fix